web3.js/src/nonce-account.ts

```diff
@@ -36,9 +35,14 @@ const NonceAccountLayout = BufferLayout.struct<
 
 export const NONCE_ACCOUNT_LENGTH = NonceAccountLayout.span;
 
+/**
+ * A durable nonce is a `sha256('DURABLE_NONCE', recentBlockhash)` encoded as a base58 string.
```

Comment:
I actually think we should be light on details on how the nonce value is constructed because it could change in the future. I think it's sufficient to say that it's a 32 byte value encoded as a base58 string

Reply:
👍🏻